packages/docs/src/pages/components/upload/examples/ExValidation.vue

```diff
@@ -12,12 +12,14 @@
     </b-field>
 </template>
 
-<script>
-    export default {
-        data() {
-            return {
-                file: null,
-            }
+<script lang="ts">
+import { defineComponent } from 'vue'
+
+export default defineComponent({
```

Comment:
Please see [this comment](https://github.com/ntohq/buefy-next/pull/269/files#r1732964052).